runtime/manifest-parser.peg

```diff
@@ -449,7 +449,7 @@ ParticleProvidedSlotItem
   / ParticleProvidedSlotView
 
 ParticleProvidedSlotView
-  = 'view' whiteSpace view:lowerIdent eolWhiteSpace
+  = ('view' / 'handle') whiteSpace view:lowerIdent eolWhiteSpace
```

Comment:
I wondered about perhaps replacing all internal-only `view` references while we're here but fine to do later.

Reply:
Yeah, we should definitely do it. I think I'll keep poking away at these smallish cleanup CLs until everything's good.